analytics/terraform/spark-k8s-operator/examples/benchmark/tpcds-benchmark-3t.yaml

```diff
@@ -1,38 +1,37 @@
 # NOTE: This example requires the following prerequisites before executing the jobs
-# 1. Ensure spark-team-a name space exists
-# 2. replace <S3_BUCKET> with your bucket name
-# 3. Ensure you run "analytics/spark-k8s-operator/spark-samples/tpcds-benchmark-data-generation-1t.yaml"  which generates 3 TB input data
+# 1. Create an S3 bucket to store the generated data
+# 2. Ensure spark-team-a name space exists
+# 3. replace <S3_BUCKET> with your bucket name
 
 ---
 apiVersion: "sparkoperator.k8s.io/v1beta2"
 kind: SparkApplication
 metadata:
   name: tpcds-benchmark-3tb
   namespace: spark-team-a
-  labels:
-    app: "tpcds-benchmark"
-    applicationId: "tpcds-benchmark-3t"
-    # Assign the job to a Yunikorn Queue via label.
-    queue: root.prod
+
 spec:
+  # batchScheduler: yunikorn
+  # batchSchedulerOptions:
+  #   queue: root.default
   type: Scala
   mode: cluster
-  image: public.ecr.aws/data-on-eks/emr-on-eks-benchmark:3.1.2
+  image: public.ecr.aws/data-on-eks/spark3.5.3-scala2.12-java17-python3-ubuntu-tpcds:v2
   imagePullPolicy: IfNotPresent
-  sparkVersion: 3.1.2
+  sparkVersion: 3.5.3
   mainClass: com.amazonaws.eks.tpcds.BenchmarkSQL
   mainApplicationFile: local:///opt/spark/examples/jars/eks-spark-benchmark-assembly-1.0.jar
   arguments:
     # TPC-DS data location
-    - "s3://blogpost-sparkoneks-us-east-1/blog/BLOG_TPCDS-TEST-3T-partitioned"
+    - "s3a://<S3_BUCKET>/TPCDS-TEST-1TB"
     # results location
-    - "s3://<S3_BUCKET>/TPCDS-TEST-3T-RESULT"
+    - "s3a://<S3_BUCKET>/TPCDS-TEST-1T-RESULT"
```

Comment:
naming nit, if we use a 1TB data set can we rename the file(s) to `tpcds-benchmark-1t.yaml`?